src/modules/iotjs_module_tls.c

```diff
@@ -246,23 +243,23 @@ JS_FUNCTION(TlsContext) {
   // User provided trusted certificates
   jerry_value_t jcert_auth =
       iotjs_jval_get_property(joptions, IOTJS_MAGIC_STRING_CA);
+  iotjs_string_t bjcert_auth = iotjs_string_create();
```

Comment:
Just like above, the `ioths_jval_get_data` will create the required strings, no need to call the `iotjs_string_create` method.

Reply:
Same as my comment above.